File: Weapon/ColdWeapon/SapperBlade.php
<?php


namespace Weapon;


class SapperBlade extends ColdWeapon
{
    private $handleLength;

    final public function getHandleLength(){
        return $this->handleLength;
    }

    final public function setHandleLength(float $handleLength){
        $this->handleLength = $handleLength;
    }
}

File: Weapon/ColdWeapon/SapperBlade.py
from ColdWeapon import ColdWeapon


class SapperBlade(ColdWeapon):
    __handleLength = None

    @property
    def handle_length(self):
        return self.__handleLength

    @handle_length.setter
    def handle_length(self, value: int):
        self.__handleLength = value
